src/test/java/uk/gov/justice/hmpps/datacompliance/repository/jpa/repository/referral/OffenderDeletionReferralRepositoryTest.java

```diff
@@ -68,12 +87,24 @@ private void assertMatchesExpectedContents(final OffenderDeletionReferral referr
         assertThat(referral.getMiddleName()).isEqualTo("Middle");
         assertThat(referral.getLastName()).isEqualTo("Smith");
         assertThat(referral.getBirthDate()).isEqualTo(LocalDate.of(1969, 1, 1));
-        assertThat(referral.getReceivedDateTime()).isEqualTo(LocalDateTime.of(2020, 1, 1, 1, 2, 3));
+        assertThat(referral.getReceivedDateTime()).isEqualTo(LocalDateTime.of(2020, 1, 2, 3, 4, 5));
 
         assertThat(referral.getReferredOffenderBookings()).hasSize(1);
-        final var offenderBooking = referral.getReferredOffenderBookings().get(0);
+        assertMatchesExpectedContents(referral.getReferredOffenderBookings().get(0));
+
+        assertMatchesExpectedContents(referral.getReferralResolution().orElseThrow());
+    }
 
+    private void assertMatchesExpectedContents(final ReferredOffenderBooking offenderBooking) {
         assertThat(offenderBooking.getOffenderId()).isEqualTo(-1001L);
         assertThat(offenderBooking.getOffenderBookId()).isEqualTo(-1L);
     }
+
+    private void assertMatchesExpectedContents(final ReferralResolution resolution) {
+        assertThat(resolution.getResolutionDateTime()).isEqualTo(LocalDateTime.of(2021, 2, 3, 4, 5, 6));
+        assertThat(resolution.getResolutionType()).isEqualTo(DELETED);
+        assertThat(resolution.isType(DELETED)).isTrue();
+        assertThat(resolution.isType(RETAINED)).isFalse();
```

Comment:
Due you need these 2 (last) assertions.  You have confirmed the type already.

Reply:
Yeah, probably unnecessary, just included for completeness